packages/orchestration/src/utils/packet.js

```diff
@@ -108,7 +108,7 @@ export function parseQueryPacket(response) {
   return harden(
     responses.map(resp => ({
       ...resp,
-      height: String(resp.index),
+      height: String(resp.height),
```

Comment:
incidentally, I'm surprised there isn't a function already to do this encoding.

Reply:
Good callout - see [1962218](https://github.com/Agoric/agoric-sdk/pull/9628/commits/1962218d7536915cbe10ef29e2efb306acc3426d)